doc/source/reference/random/index.rst

```diff
@@ -151,11 +151,6 @@ What's New or Different
   select distributions
 * Optional ``out`` argument that allows existing arrays to be filled for
   select distributions
-* All BitGenerators can produce doubles, uint64s and uint32s via CTypes
```

Comment:
This statement is still true on 1.17.x.

Reply:
@rgommers Comment?

Reply:
no idea, I have no reason to doubt @bashtage's comment

Reply:
Should we omit the deletion here, or is reasonable for 1.17 and we will cover things better in 1.18? I'd like to put this backport in. Maybe have another if there are changes in master?

Reply:
Happy to omit this deletion, will verify later today or tomorrow and change if it still works.

Reply:
OK, will wait.  @bashtage makes the same comment in several places.

Reply:
@rgommers Still want to look at this?

Reply:
sorry, failed to make time - not going to happen until end of next week at the earliest. added the 1.17.4 milestone, it seems that it should go into that release

Reply:
reverted the deletion: `BitGenerator.ctypes` still exists